docs/MANPAGE_SYNTAX.md

```diff
@@ -125,3 +136,5 @@ Normally, the dates of changes, the content of the changes and the person who pr
 
 Example:\
 December 2021, Originally compiled by Alexander Richter <example@redhat.com>
+
+`A new line is needed of the end of every manpage.`
```

Comment:
I like the thought, but it might cause havoc in some of our CI systems under the Containers umbrella.  @mheon or @edsantiago, do you know if the Podman CI would spit up with a blank line at the end of a man page?  I think so, but don't know for a fact.

Reply:
I don't mean that a new line is needed after the line break of the last "topic", I just wanted to clarify that at the end of every manpage there should be an empty line.
I will rephrase it.